web/locales/de.json

```diff
@@ -93,7 +93,7 @@
         "consumption": "Verbrauch",
         "cpinfo": "<b>Verbrauch</b> bezieht Importe und Exporte mit ein, <b>Produktion</b> ignoriert sie",
         "selectLanguage": "Sprache auswählen",
-        "lowCarbDescription": "Beinhaltet regenerative Energie and Atomkraft"
+        "lowCarbDescription": "Beinhaltet regenerative Energien and Kernenergie"
```

Comment:
```suggestion
        "lowCarbDescription": "Beinhaltet regenerative Energien und Kernenergie"
```

typo leftover

Reply:
Yes, this was the old PullRequest. But it was closed because I didn't see the sesponse....